code/io/increaseHumanGEMVersion.m

```diff
@@ -42,23 +41,37 @@ function increaseHumanGEMVersion(bumpType)
 newVersion = num2str(newVersion,'%d.%d.%d');
 
 %Check if history has been updated:
-%fid     = fopen('../../history.md','r');
-%history = fscanf(fid,'%s');
-%fclose(fid);
-%if ~contains(history,['human' newVersion ':'])
-%    error('ERROR: update history.md first')
-%end
-%To be included
+fid     = fopen(fullfile(modelPath,'history.md'),'r');
+history = fscanf(fid,'%s');
+fclose(fid);
+if ~contains(history,['human' newVersion ':'])
+    error('ERROR: update history.md first')
+end
 
 %Load model:
-ymlFile=fullfile(modelPath,'model','Human-GEM.yml');
-ihuman = importYaml(ymlFile);
+ihuman = readYAMLmodel(fullfile(modelPath,'model','Human-GEM.yml'));
 
 %Include tag and save model:
 ihuman.version = newVersion;
 
-%Export model to multiple formats
-exportHumanGEM(ihuman,'Human-GEM',modelPath,{'mat', 'txt', 'xml', 'yml', 'xlsx'});
+%Check if it matches reactions.tsv, metabolites.tsv and genes.tsv
+fields = {'rxns','reactions';'mets','metabolites';'genes','genes.tsv'};
+for i=1:size(fields,1)
+    tsvList = importTsvFile(fullfile(modelPath,'model',[fields{i,2} '.tsv']));
+    Lia     = ismember(ihuman.(fields{i,1}), tsvList.(fields{i,1}));
+    dispEM(['The following ' fields{i,2} ' are in model.' fields{i,1} ...
+            ' but not in model/' fields{i,2} '.tsv:'],true,ihuman.(fields{i,1})(~Lia));
+    Lia     = ismember(tsvList.(fields{i,1}), ihuman.(fields{i,1}));
+    dispEM(['The following ' fields{i,2} ' are in model/' fields{i,2} ...
+            '.tsv but not in model.' fields{i,1} ':'],true,tsvList.(fields{i,1})(~Lia));
+end
+
+%Export model to multiple formats, without annotation
+writeYAMLmodel(ihuman,fullfile(modelPath,'model','Human-GEM.yml'),false,false);
```

Comment:
also the I/O cycle reports an error.
Error using fprintf
Function is not defined for 'cell' inputs.

Error in writeYAMLmodel>writeField (line 279)
            fprintf(fid,'    %s: %s\n',name,list);

Error in writeYAMLmodel (line 101)
    writeField(model, fid, 'eccodes',              'txt', i, '  - eccodes',               preserveQuotes)